packages/account/src/Styles/account.scss

```diff
@@ -1392,7 +1393,7 @@ $MIN_HEIGHT_FLOATING: calc(
     }
     @include desktop {
         .table__body .table__row:last-child {
-            border-bottom: 1px solid var(--general-section-1);
+            border-bottom: none;
```

Comment:
Why do we remove the border? It seems needed.

Reply:
As I noticed in production, there is no border under the last row:
<img width="1780" alt="Screenshot 2021-07-28 at 09 07 16" src="https://user-images.githubusercontent.com/87321516/127272422-277309aa-3686-4126-aa52-d5bd1cc65d13.png">
![image](https://user-images.githubusercontent.com/87321516/127272495-411894da-4ee8-4dea-83a9-823d944a7ed8.png)
And we also have the same code fragment in data-table.scss (line 74) where the border is explicitly removed:
`.table`...
    `&__body`...
  **line 74:**   `.table__row:last-child {
                               border-bottom: none;
                          }`
Should I revert the change to make the border visible?

Reply:
It actually adds a border below every row, and the style in `data-table.scss` removes it from the last row only. Currently, your change removes the border between the rows, so I think we better revert it.
![image](https://user-images.githubusercontent.com/71861519/127274184-3111294d-0a30-4a7d-80a1-1efb8d321faa.png)

Reply:
got it, thank you!